packages/builder/src/components/backend/DataTable/TableHeader/TableHeader.svelte

```diff
@@ -103,6 +103,15 @@
     opacity: 1;
   }
 
+  .column-header-name {
+    white-space: normal !important;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 1;
+    overflow: hidden;
```

Comment:
NAB: is white-space normal combined with the webkit prefix styles any different than just doing white-space nowrap?

Reply:
I attempted no-wrap earlier and it didn't work. I eventually got to this and it worked so I just pushed it.

Reply:
@aptkingston I tried your advice, but it didn't work. I'm possibly doing something wrong.

Reply:
Still need the rest of the styles you had though - so nowrap + overflow hidden + text-overflow ellipsis is what I've always used for wrapping. Maybe due to other reasons it needs a width specified or something, so if it doesn't work here then stick with what you've got. Had just never seen that particular combo of styles to achieve overflow before but it's good to know.